website/source/docs/guides/leader-election.html.md

```diff
@@ -1,130 +1,141 @@
 
--> **Note:** JSON output in this guide has been pretty-printed for easier reading. Actual values returned from the API will not be formatted.
+If you
+are interested in the leader election used internally by Consul, please refer to the
+[consensus protocol](/docs/internals/consensus.html) documentation instead.
 
-## Contending Nodes
+## Contending Service Instances 
 
-Let's imagine we have a set of nodes who are attempting to acquire leadership
-for a given service. All nodes that are participating should agree on a given
+Imagine you have a set of service instances who are attempting to acquire leadership
+for a given service. All service instances that are participating should agree on a given
 key to coordinate. A good pattern is simply:
 
 ```text
 service/<service name>/leader
 ```
 
-We'll abbreviate this pattern as simply `<key>` for the rest of this guide.
+Let's abbreviate this pattern as simply as a `<key>` for the rest of this guide.
+
+```text
+service/dbservice/leader = lead
+```
+
+### Create a Session 
 
 The first step is to create a session using the
-[Session HTTP API](/api/session.html#session_create):
+[Session HTTP API](/api/session.html#session_create).
 
-```text
-curl  -X PUT -d '{"Name": "dbservice"}' \
-  http://localhost:8500/v1/session/create
- ```
+```sh
+$ curl  -X PUT -d '{"Name": "dbservice"}' http://localhost:8500/v1/session/create
+```
 
 This will return a JSON object containing the session ID:
 
-```text
+```json
 {
   "ID": "4ca8e74b-6350-7587-addf-a18084928f3c"
 }
 ```
 
-The next step is to acquire a session for a given key from this node
+### Acquire a Session 
+
+The next step is to acquire a session for a given key from this instance
 using the PUT method on a [KV entry](/api/kv.html) with the
-`?acquire=<session>` query parameter. The `<body>` of the PUT should be a
-JSON object representing the local node. This value is opaque to
+`?acquire=<session>` query parameter. 
+
+The `<body>` of the PUT should be a
+JSON object representing the local instance. This value is opaque to
 Consul, but it should contain whatever information clients require to
 communicate with your application (e.g., it could be a JSON object
 that contains the node's name and the application's port).
 
-Attempt to `acquire` the `<key>`. This will look something like (note that
-`<session>` is the ID returned by the call to
-[`/v1/session/create`](/api/session.html#session_create)):
-
-```text
-curl -X PUT -d <body> http://localhost:8500/v1/kv/<key>?acquire=<session>
+```sh
+$ curl -X PUT -d <body> http://localhost:8500/v1/kv/lead?acquire=ca8e74b-6350-7587-addf-a18084928f3c
  ```
 
 This will either return `true` or `false`. If `true`, the lock has been acquired and
-the local node is now the leader. If `false` is returned, some other node has acquired
+the local service instance is now the leader. If `false` is returned, some other node has acquired
```

Comment:
The reference to "node" here could be removed in favor of "service" for consistency:

> If `true`, the lock has been acquired and the local node is now the leader. If `false` is returned, some other instance has acquired the lock.

Reply:
Fixed a typo in my suggestion ^